api/views/maps.py

```diff
@@ -0,0 +1,56 @@
+from api import app, db
+from api.models import Map, POI
+from api.utils import create_response, row_constructor
+from flask import Blueprint, request, jsonify
+import json
+#from api.utils import InvalidUsage
+# from sqlalchemy import func
+# import time
+# from datetime import date
+# import uuid
+# from flask_sqlalchemy import SQLAlchemy
+
+MAPS_URL = '/maps'
+MAPS_ID_URL = '/maps/<int:map_id>'
+
+mod = Blueprint('maps', __name__)
+
+#none of these have been comprehsnsively tested yet
+
+# Get all maps.
+@app.route(MAPS_URL, methods = ['GET'])
+def get_map_years():
+    maps = [m.to_dict() for m in Map.query.all()]
+    return create_response({'maps' : maps})
+
+# Post a Map
+@app.route(MAPS_URL, methods = ['POST'])
+def create_map():
+    data = request.get_json()
+    fields = ['image_url', 'map_year']
+    map_info = [data.get(field) for field in fields]
+    if None in map_info:
+        missing_params = filter(lambda x: data.get(x) == None, fields)
+        message = 'Missing parameters ' + ', '.join(missing_params)
+        return create_response(data, 400, message)
+    else:
```

Comment:
you dont need this `else` clause since you `return` in the `if`. im personally a fan of minimizing code flow branches since it makes it less complex